roles/openshift_cfme/defaults/main.yml

```diff
@@ -28,6 +28,7 @@ openshift_cfme_pv_data:
 openshift_cfme_maxImagesBulkImportedPerRepository: 100
 # Hostname/IP of the NFS server. Currently defaults to first master
 openshift_cfme_nfs_server: "{{ groups.nfs.0 }}"
+openshift_cfme_nfs_directory: "/exports"
```

Comment:
Please add description here to help users understand the implications of this.

The README may also need to be updated to reflect the fact that this is configurable and how it works. So maybe comments in two places are required.